internal/datastore/datastore.go

```diff
@@ -14,8 +14,9 @@ const Ellipsis = "..."
 
 // RevisionChanges represents the changes in a single transaction.
 type RevisionChanges struct {
-	Revision Revision
-	Changes  []*v0.RelationTupleUpdate
+	Revision            Revision
+	Changes             []*v0.RelationTupleUpdate
+	RelationshipUpdates []*v1.RelationshipUpdate
```

Comment:
I favored adding this here instead of converting all `v0.RelationTupleUpdate` objects downstream because it makes downstream consumption easier. At one point when we deprecate v0 officially we can just drop the `Changes` field altogether and refactor the datastore implementations to omit it as well.

Reply:
I have a slight concern about performance, but as you said, we can just deprecate v0 at some point and convert in there if need be